testcontainers/src/clients/http.rs

```diff
@@ -372,6 +375,18 @@ mod tests {
         assert_eq!(publish_ports, true, "publish_all_ports must be `true`");
     }
 
+    #[tokio::test(flavor = "multi_thread")]
+    async fn api_run_command_should_map_exposed_port() {
+        let docker = Http::new();
+        let image = GenericImage::new("simple_web_server", "latest").with_exposed_port(5000);
+        let container = docker.run(image).await;
+
+        // inspect volume and env
+        let container_details = inspect(&docker.inner.bollard, container.id()).await;
+        let ports = container_details.network_settings.unwrap().ports.unwrap();
+        assert!(ports.contains_key("5000/tcp"), "{:?}", ports);
```

Comment:
Would it perhaps make more sense to use the public API of the container to test this?

Reply:
Yes that makes far more sense. I've amended the test :)